openedx/core/djangoapps/timed_notification/tasks.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+from datetime import datetime, timedelta
+
+from core import send_course_notification_email, _get_course_link
+
+from celery.task import periodic_task
+from celery.schedules import crontab
+from lms.djangoapps.branding import get_visible_courses
+from pytz import utc
+from django.db import close_old_connections
+
+log = logging.getLogger('edx.celery.task')
+
+
+@periodic_task(run_every=crontab(minute=1, hour=0))
+def execute_dialy():
+    log.info("TIMED-NOTIFICATION-JOB execute_dialy")
+
+    close_old_connections()
```

Comment:
what is the reason for this statement?

Reply:
without this statement I'm facing "MySQL server has gone away" exception. I found this error in the following link:
https://dev.mysql.com/doc/refman/5.7/en/gone-away.html

then I found the "db.close_old_connections()" against this problem, on this page. https://code.djangoproject.com/ticket/21597